.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Isn't there a way to configure the source path for prettier to use, so we don't need to say the path when invoking it?

Might be worth checking the documentation imo

Reply:
I think I need to change this in the package.json: "prettier:fix": "prettier --write src", but I don't know "how" yet. Just removing "src" doesn't work :D

Reply:
To remove the path, in this case `src`, we need to add it to prettier configuration, if such an option exists. The documentation only mention that we can [ignore code](https://prettier.io/docs/en/ignore).

Maybe there is nothing we can do, and the paths must be kept in the command line.

Reply:
It seems that in the command line, we always need to specify the path. Can I finish this conversation?